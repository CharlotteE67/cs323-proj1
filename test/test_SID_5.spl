int test_func_1({
    int temp;
    temp = '2';
}


int main(){

}