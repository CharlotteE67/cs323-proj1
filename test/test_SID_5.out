Error type B at Line 0: Missing right parenthesis ')'
Error type A at Line 8: hexadecimal int with illegal char
Error type B at Line 8: Missing comma ','
Error type B at Line 11: Missing right parenthesis ')'
Error type B at Line 19: Missing right parenthesis ')'
